File: chroma-league/src/test/java/com/bonepl/chromaleague/hud/animations/AllyElderDragonKillAnimationTest.java
package com.bonepl.chromaleague.hud.animations;

import com.bonepl.chromaleague.hud.AnimationTester;
import org.junit.jupiter.api.Test;

class AllyElderDragonKillAnimationTest {
    @SuppressWarnings("JUnitTestMethodWithNoAssertions")
    @Test
    void playAllyElderDragonKillAnimation() {
        new AnimationTester().testAnimation(new AllyElderDragonKillAnimation());
    }
}

File: test/hud/animations/AllyElderDragonKillAnimation.test.ts
import {describe, test} from 'vitest';
import {AnimationTester} from '../../AnimationTester.js';
import {AllyElderDragonKillAnimation} from '../../../src/hud/animations/AllyElderDragonKillAnimation.js';

/**
 * Integration test for ally elder dragon kill animation on actual hardware.
 */
describe('AllyElderDragonKillAnimation Integration', () => {
    test('playAllyElderDragonKillAnimation', async () => {
        await new AnimationTester().testAnimation(new AllyElderDragonKillAnimation());
    }, 30000);
});
